install_clam.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+apt-get update -qq
+apt-get install -qq clamav-daemon
+echo TCPSocket 3310 >> /etc/clamav/clamd.conf
```

Comment:
ok I was trying to do a bit more complicated logic to get around the time it was taking to download the virus definitions but it wasn't working due to stupid permission problems and the slowness went away. Builds were taking up to 20min!